command/server/config.go

```diff
@@ -112,6 +112,8 @@ type Config struct {
 
 	DetectDeadlocks string `hcl:"detect_deadlocks"`
 
+	DisableLeaseRoleTracking bool `hcl:"disable_lease_role_tracking"`
```

Comment:
It's going to be hard to find a good name for this option.  We're not actually disabling lease role tracking altogether, right?  We're just breaking the ability to get correct lease counts for role quotas that are created subsequently.

Reply:
🤔 Yeah, this is a good point and probably worthy of a call-out in the conditional. We'll still be setting the `LoginRole` for leases AFTER a role-based quota is set on the mount.

I've written down and deleted 4 names so far, so I concur, very difficult to name. The best I can come up with is something like:

`CountOldLeasesForNewRoleQuotas` (default: true)

but even that seems kind of awful.

Reply:
I'm fine with that.  Could also do something like `ImpreciseLeaseRoleTracking`.